sdks/java/io/google-cloud-platform/src/main/java/org/apache/beam/sdk/io/gcp/bigquery/BigQueryServicesImpl.java

```diff
@@ -842,21 +844,25 @@ public void deleteDataset(String projectId, String datasetId)
       }
 
       long retTotalDataSize = 0;
-      List<TableDataInsertAllResponse.InsertErrors> allErrors = new ArrayList<>();
+      List<TableDataInsertAllResponse.InsertErrors> rowsFailedToInsertErrors = new ArrayList<>();
       // These lists contain the rows to publish. Initially the contain the entire list.
       // If there are failures, they will contain only the failed rows to be retried.
       List<FailsafeValueInSingleWindow<TableRow, TableRow>> rowsToPublish = rowList;
       List<String> idsToPublish = null;
+
       if (!ignoreInsertIds) {
         idsToPublish = insertIdList;
       }
+
+      List<InsertAttemptParams> insertAttemptParamsForEachRequest = new ArrayList<>();
```

Comment:
This change looks irrelevant to the PR description. Would it be better to split into a separate PR?